physics/gravitational_force.cpp

```diff
@@ -0,0 +1,36 @@
+/**
+ * @file gravitational_force.cpp
+ * @brief
+ * https://en.wikipedia.org/wiki/Newton%27s_law_of_universal_gravitation
+ * @details
+ *newton's law of universal gravitation
+ * @author [Ayaanlehashi11](https://github.com/ayaanlehashi11)
+ * @see
+ */
+
+#include <iostream>
+#include <cmath>
+#include <cstdint>
```

Comment:
Please add a one-line description of what the library/header is for (**see the example below**).

```c++
#include <iostream>    /// for IO operations
#include <cassert>    /// for assert
```